browser/resources/settings/brave_overrides/privacy_page.ts

```diff
@@ -181,6 +181,36 @@ function InsertShieldsSubpage (pages: Element)
     `)
 }
 
+function InsertBraveAIChatSubpage (pages: Element)
+{
+  pages.appendChild(
+    html`
+      <template is="dom-if" route-path="/content/braveAIChat" no-search>
+        <settings-subpage
+          associated-control="[[$$('#braveAIChat')]]"
+          page-title="${loadTimeData.getString('siteSettingsBraveAIChat')}">
+          <settings-category-default-radio-group
+              id="braveAIChatDefault"
+              category="[[contentSettingsTypesEnum_.BRAVE_AI_CHAT]]"
+              block-option-label=
+                "${loadTimeData.getString('siteSettingsBraveAIChatBlock')}"
+              allow-option-label=
+                "${loadTimeData.getString('siteSettingsBraveAIChatAsk')}"
```

Comment:
huh, not your problem but its weird we do this rather than using the `$i18n` util. I guess there's probably a reason :laughing: